Apps/APN/package.json

```diff
@@ -21,20 +21,30 @@
     "expo-splash-screen": "~0.11.2",
     "expo-status-bar": "~1.0.4",
     "expo-updates": "~0.8.1",
-    "react": "18.0.0",
+    "react": "18.2.0",
     "react-dom": "16.13.1",
-    "react-native": "0.69.1",
+    "react-native": "0.72.4",
     "react-native-device-info": "^10.7.0",
     "react-native-gesture-handler": "~1.10.2",
```

Comment:
Can you please update this too? Because I often had to delete `node_modules` folder from one app when running the other for `pod install` because of version mismatch 😅

Reply:
Thanks for letting me know, Rehan! I have updated the library - `react-native-getsure-handler`